addons/optionsmenu/gui/settingsMenu.hpp

```diff
@@ -277,7 +277,7 @@ class ACE_settingsMenu {
 };
 class ACE_serverSettingsMenu: ACE_settingsMenu {
     onLoad = QUOTE(uiNamespace setVariable [ARR_2('ACE_serverSettingsMenu', _this select 0)]; [] call FUNC(onServerSettingsMenuOpen););
-    onUnload = QUOTE(uiNamespace setVariable [ARR_2('ACE_serverSettingsMenu', nil)];);
+    onUnload = QUOTE(uiNamespace setVariable [ARR_2('ACE_serverSettingsMenu', nil)]; [] call FUNC(onSettingsMenuOpen););
```

Comment:
The original setting menu is still open when we close the server menu, and it acts weird.  We could also just close it when we open the server settings.